domdiv/draw.py

```diff
@@ -922,6 +930,7 @@ def drawText(self, card, divider_text="card", wrapper="no"):
                     dmod = d
                 else:
                     dmod = self.add_inline_images(d, s.fontSize)
+                print('PARA ({}): "{}"'.format(card.name, dmod))
```

Comment:
Not sure what this new print statement is doing. Not saying it shouldn't be here it just isn't replacing anything obvious in the diff and warrants a double check.